www/status.php

```diff
@@ -22,15 +16,12 @@
 
 $shownServices = [];
 
-
 if (empty($shownServicesList)) {
     $shownServices = $services;
 } else {
-    foreach ($services as $service) {
-        $serviceName = $service['name'];
-        if (in_array($serviceName, array_keys($shownServicesList))) {
-            $service['name'] = $shownServicesList[$serviceName]['name'];
-            $service['description'] = $shownServicesList[$serviceName]['description'];
+    foreach ($shownServicesList as $service) {
+        if (isset($services[$service['host']][$service['probe_identifier']])) {
```

Comment:
Oh, this getting of property is crazy. Please just fetch the partial data like $host = $service['host'] and then use it as keys to improve the readability.